### ISSUE TITLE
Invalid syntax in fritzbox_helper.py

### ISSUE THREAD

**Issue by Yogi4301** [Tafkas/fritzbox-munin#54]
Hello,

many thanks for adapting the plugins to Fritz!OS version 7.25!

Actually I got the following plugins to run with some of the changes you did - fritzconnection has actually version 0.85 (the highest available version on RPi):
fritzbox_connection_uptime.py
fritzbox_traffic.py

But now on the other plugins I get the following error in the f-statements
```

Traceback (most recent call last):
  File "/etc/munin/plugins/fritzbox_cpu_usage.py", line 22, in <module>
    import fritzbox_helper as fh
  File "/usr/share/munin/plugins/fritzbox_helper.py", line 93
    return f"{challenge_parts[4]}${hash2.hex()}"
                                               ^
SyntaxError: invalid syntax
```
My python version is 3.5.3 - is it possible to replace the f-statements ?

**Comment by Yogi4301** [comment]
In the next step I bypassed the PBKDF2 stuff by removing 'version=2' from the url but then I ran in the next problem. The statement   json.loads(xhr_data) does not word, because the XHR_DATA beginns with a 'b' ......
Is it possible to provide a version of the fritzbox_helper.y compatible with python2?

**Comment by Yogi4301** [comment]
Actually I use a workaround and removed 'version=2' from the url - and with some minor changes I switched to python2. But how do I have to write the above statment?

**Comment by Tafkas** [comment]
What you could do is:

`return "{}${}".format(challenge_parts[4], hash2.hex())`

**Comment by Tafkas** [comment]
I've fixed it and created a new tag `7.25.1`

**Comment by Yogi4301** [comment]
Thanks, but now I get the folling error in fritzbox_cpu_temperature.py-maybe also a python 2 problem. Without 'version=2' all plugins are working now.

```
Traceback (most recent call last):
  File "/etc/munin/plugins/fritzbox_cpu_temperature.py", line 62, in <module>
    get_cpu_temperature()
  File "/etc/munin/plugins/fritzbox_cpu_temperature.py", line 34, in get_cpu_temperature
    session_id = fh.get_session_id(server, username, password)
  File "/usr/share/munin/plugins/fritzbox_helper.py", line 55, in get_session_id
    challenge_response = calculate_pbkdf2_response(state.challenge, password)
  File "/usr/share/munin/plugins/fritzbox_helper.py", line 86, in calculate_pbkdf2_response
    salt1 = bytes.fromhex(challenge_parts[2])
AttributeError: type object 'str' has no attribute 'fromhex'
```
